.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,25 @@
+---
+    name: pre-commit
+
+    on:
+      pull_request:
+
+    jobs:
+      lint-pr-title:
+        name: Validate PR titles
+        runs-on: ubuntu-latest
+        steps:
+        - uses: amannn/action-semantic-pull-request@v5
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      pre-commit:
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v3
+        - uses: actions/setup-python@v4.6.1
+          with:
+            python-version: '3.8'
+        - run: pip install 'virtualenv<20.11.0'
```

Comment:
some of the precommit hooks needed it

Reply:
As long as it's because it's because of the specific things added for a Python project that's fine.

FWIW, I've already been pretty dubious on the value of precommit. If people want to use it, fine, but trying to use it in CI to guarantee parity has been miserable. 

No need to change anything here but when I have time to work on Terraform workflows I ultimately plan to sever pre-commit from Actions entirely and not have any continuous testing of the pre-commit configs.